include/net/net_stats.h

```diff
@@ -225,6 +233,7 @@ struct net_stats_tc {
 		net_stats_t pkts;
 		net_stats_t bytes;
 		u8_t priority;
+		struct net_stats_rx_time rx_time;
```

Comment:
you could probably reorder, to put priority at the end. (looks like it's same in sent array.